tests/unit/utils/test_pipeline_docker_image_builder.py

```diff
@@ -38,6 +38,28 @@ def test_including_active_profile_in_build_context(tmp_path: Path):
     assert not config_path.exists()
 
 
+def test_check_user_is_set(tmp_path: Path):
```

Comment:
```suggestion
def test_check_user_is_set():
```
Don't think the `tmp_path` is needed here

Reply:
Agreed. Removed